sympy/functions/combinatorial/factorials.py

```diff
@@ -862,8 +862,9 @@ def eval(cls, n, k):
             if n.is_integer and n.is_nonnegative and d.is_negative:
                 return S.Zero
             if n.is_number:
-                from sympy.simplify.simplify import simplify
-                return simplify(cls._eval(n, k))
+                from sympy.core.function import expand
```

Comment:
It is not necessary to import expand as it is a method.

Reply:
Sorry!